src/main/java/org/apache/commons/csv/CSVRecord.java

```diff
@@ -211,7 +211,18 @@ public boolean isMapped(final String name) {
      * @return whether a given columns is mapped and has a value
      */
     public boolean isSet(final String name) {
-        return isMapped(name) && getHeaderMapRaw().get(name).intValue() < values.length;
+        return isMapped(name) && getHeaderMapRaw().get(name) < values.length;
```

Comment:
Let's keep the boxing as is please and focus this PR on the new API. 

If you want to go through the whole code base and consistently use one boxing style that's a different PR IMO. But: I prefer to know when boxing and unboxing takes place since it is not free; this gives the compiler the opportunity to flag unintentional boxing and unboxing and let the programmer decide that's really what should happen.

Reply:
Ok, I've rolled back this change.